profiles/target/2.0/datapackage.json

```diff
@@ -57,7 +57,7 @@
     "homepage": {
       "propertyOrder": 60,
       "title": "Home Page",
-      "description": "The home on the web that is related to this data package.",
+      "description": "The home on the web that is related to this resource.",
```

Comment:
Two notes:

1. These files should be rebuild based on the changes above.
2. Is it a good idea to rebuild these files? I think we should have a new directory `target/2.1-draft`

Reply:
I am not familiar with the build process, in particular what is the difference between "build/profiles" and "profiles/target" ?

When I run the "build.js", with "VERSION=2.1-draft", it does not create (or for the matter, populate) a `target/2.1-draft` directory :thinking:

Reply:
@peterdesmet 
@pierrecamilleri 
I need to set up the branching process for new versions so it's OK if this PR just includes actual changes to the standard and I'll handle the rest